test/tools/ossfuzz/SolidityMutator.h

```diff
@@ -0,0 +1,76 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include <test/tools/ossfuzz/SolidityBaseVisitor.h>
```

Comment:
Is this file autogenerated? I can't find it in the repo but tests pass on this PR so it must be available somehow...

Reply:
Yes. These lines auto generate the antlr visitors the mutator inherits: https://github.com/ethereum/solidity/blob/9ced804236153e5202a6b77b5946cf4840fe526a/scripts/ci/build_ossfuzz.sh#L35-L37

The said lines are invoked by the ossfuzz build CI.